tests/protractor/e2e/medic-collect.js

```diff
@@ -0,0 +1,76 @@
+const assert = require('chai').assert,
+    net = require('net'),
+    host = 'localhost',
+    port = '5998',
+    dbName = 'medic';
```

Comment:
As you pointed out, you should be able to get this from auth.js, right? I know it's horrible magic but I want to keep all the horrible magic in one place...